mungegithub/mungers/sig-mention-handler_test.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"reflect"
+	"runtime"
+	"testing"
+
+	githubapi "github.com/google/go-github/github"
+	github "k8s.io/test-infra/mungegithub/github"
+)
+
+const (
+	helpWanted = "help-wanted"
+	open       = "open"
+	sigApps    = "sig/apps"
+	username   = "Ali"
+)
+
+func TestSigMentionHandler(t *testing.T) {
+	runtime.GOMAXPROCS(runtime.NumCPU())
+
+	tests := []struct {
+		name     string
+		obj      *github.MungeObject
+		expected bool
+	}{
+		{
+			name: "is a pull request",
```

Comment:
I would prefer we specify the results in addition to the setup so `name: "ignore PRs"`, which seems easier to understand to me